connectors/src/connectors/microsoft/temporal/file.ts

```diff
@@ -231,9 +228,8 @@ export async function syncOneFile({
         );
       }
     }
-  } else if (mimeType === "application/pdf") {
-    const data = Buffer.from(downloadRes.data);
-    documentSection = await handleTextExtraction(data, localLogger, file);
+  } else if (mimeType === "text/plain") {
+    documentSection = handleTextFile(downloadRes.data, maxDocumentLen);
   } else {
```

Comment:
getDocumentContent/mammoth only support doc file, we should test the type here, and eventually also move that part to shared/file ?